package.json

```diff
@@ -3,14 +3,14 @@
   "version": "0.0.2",
   "private": true,
   "dependencies": {
-    "@measured/puck": "^0.9.0",
-    "@measured/puck-plugin-heading-analyzer": "^0.9.0",
+    "@measured/puck": "0.12.0-canary.c066397",
+    "@measured/puck-plugin-heading-analyzer": "0.12.0-canary.c066397",
```

Comment:
This could be set back to `"^0.12.0"` now

Reply:
Done!